pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -66,6 +66,8 @@ type PipelineRunSpec struct {
 	// with those declared in the pipeline.
 	// +optional
 	Workspaces []WorkspaceBinding `json:"workspaces,omitempty"`
+	// +optional
```

Comment:
could use a comment here describing the field's purpose.  `// LimitRangeName specifies the limit range to use for minimum resource values in task pods` or similar.

Same comment for taskrun_types in v1alpha1 and 2

Reply:
Thanks! Added along with some docs for pipelineruns/taskruns.

Reply:
:+1: nice